app.js

```diff
@@ -44,6 +47,12 @@ const checkError = (ctx, response) => {
     }
 };
 
+const ensureTrue = (boolValue) => {
+    if (!boolValue) {
+        ctx.throw(400, `Internal error`);
```

Comment:
Internal error is `500`. `400` has specific meaning:
https://httpstatuses.com/400

Reply:
removed function